packages/typescript/src/drivers/AppiumDriver.ts

```diff
@@ -18,12 +18,16 @@ export class AppiumDriver extends BaseDriver {
   ]);
   public autoswitchContexts: boolean = true;
   public delay: number = 0;
+  public doubleFetchPageSource: boolean = false;
```

Comment:
I can't find where it's set to `true`? I understand it's a workaround, but I can't get where you use it

Reply:
It's only used in python example (environment.py#174). I added it to typescript too for API parity.